### DISCUSSION TITLE
Here's my feedback.... as a webdev of 30 years who's never used Vue, but use plenty of other component systems...

### DISCUSSION THREAD

**Discussion by davidbuzz** [oslabs-beta/PreVue3.0#42] [General]
I downloaded the code[why 2 repos?], 
figured out how to run it[seems to need a local mongo instance?], 
got it running[after following the hinds from an online chat, not the readme]?
tried to follow the README[ and couldn't, it assumes too many things] ?
Couldn't work out the right sequence of steps to make a new view or new component, it seems u need to type a name in a field, press enter, and drag things around, but I have no idea what order these thing go in, and I ended up with 10 rando boxes that i couldn't figure out how to "place" so they stayed there, they either don't move at all, or are tied to my mouse movements for ever.[need a good video, showing every mouse click, and what they are trying to do? ]
 - i suspect the app is great, but I've got no-idea, even after using it.   new-user exploration and usability 0 out of 10. 
 - I'm done for now, will come back in a year when its better.  
 - rm -rf ~/PreVue3.0/

**Comment by greenteaisgreat** [comment]
Hey there David, thank you so much for giving us a shot! Currently, we're still in the process of trying to host this project using AWS Elastic-Beanstalk and it's giving us quite a few headaches trying to accomplish that. I apologize for the lack of any intuitive method to get this spun up on one's machine; currently, a .env file would be needed to successfully spin up the backend. Hopefully once it's deployed, this aspect will be a non-issue.

I really appreciate your feedback and will make note of your responses when we iterate some more on this. You've left some great pain points for us to go off of. Thank you again for taking the time to write this out and getting back to us!
